src/api/v4/client.js

```diff
@@ -325,6 +325,53 @@ Client.prototype.getGoogleMapsMapType = function (map) {
   return this._gmapsMapType;
 };
 
+/**
+ * Return a {@link https://github.com/mapbox/tilejson-spec|TileJSON} for the map tileset.
+ *
+ * @example
+ * // Get the tilejson from the client
+ * const tilejson = client.getTilejson();
+ *
+ * @param {string} - The tileset format: vector or raster.
+ *
+ * @returns A {@link https://github.com/mapbox/tilejson-spec|L.TileJSON} for the raster or vector tileset.
+ *
+ * @api
+ */
+Client.prototype.getTilejson = function (format, layerFilters) {
```

Comment:
@IagoLast, I think your comment is gone after I fixed the linting errors.

> Do we really want to expose this function? I think is not consistent with our current raster api where we directly expose a raster layer.
> `client.getLeafletLayer().addTo(map) // raster`

I'm willing to consider any alternative that enables using any mapping library that we don't support *natively* in CARTO.js.

Reply:
Are we going to allow "clustering options" in raster layers?